dashboard-config_postgres.yaml

```diff
@@ -0,0 +1,20 @@
+  dashboard:
```

Comment:
Yes, i wanted to extract these out to env variables for Heroku any ways. Will do that..